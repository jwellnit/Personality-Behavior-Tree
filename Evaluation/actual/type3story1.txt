plan 6
Tom::Effect Text: Gather poisonous herbs and make poison.
Alchemist::Effect Text:
Guard::Effect Text: Wait.
continue? (y/n): y
plan 6
Tom::Effect Text: Gather herbs.
Alchemist::Effect Text:
Guard::Effect Text: Wait.
continue? (y/n): y
plan 6
Tom::Effect Text: Move to the Town.
Alchemist::Effect Text:
Guard::Effect Text: Wait.
continue? (y/n): y
plan 6
Tom::Effect Text: Tom knocks on the Alchemist's door.
Alchemist::Effect Text: The Alchemist answers the door and lets Tom in.
Guard::Effect Text: Wait.
continue? (y/n): y
plan 6
Tom::Effect Text: Give the herbs to the Alchemist.
Alchemist::Effect Text: Make the potion.
Guard::Effect Text: Wait.
continue? (y/n): y
plan 6
Tom is caught stealing poisoning the Alchemist.
Tom::Effect Text: Wait.
Alchemist::Effect Text: Call the guards.
Guard::Effect Text: Pursue Tom.
continue? (y/n): y
plan 6
Tom is caught stealing poisoning the Alchemist.
plan 8
Tom::Effect Text: Wait.
Alchemist::Effect Text: Call the guards.
Guard::Effect Text: Arrest Tom.
continue? (y/n):