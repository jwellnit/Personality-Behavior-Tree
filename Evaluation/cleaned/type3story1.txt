Tom collects poisonous herbs from the cottage and makes poison.
Tom collects the medicinal herbs from the cottage.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom gives the herbs to the alchemist to make medicine.
The alchemist makes the medicine using the herbs.
Tom steals the medicine from the alchemist.
The alchemist calls for the guard.
Tom steals the medicine from the alchemist.
The alchemist calls for the guard.
The guard arrests Tom.