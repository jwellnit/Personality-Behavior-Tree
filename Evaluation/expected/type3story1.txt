Tom collects the medicinal herbs from the cottage.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
Tom changes his mind.
Tom goes to the town.
Tom gives the herbs to the alchemist to make medicine.
The alchemist comes back to the shop.
Tom knocks on the door and the alchemist lets him into the shop.
The alchemist makes the medicine using the herbs.
Tom buys the medicine with a silver coin.
